<launch>

  <!--group ns="stereo"-->

    <node name="e_cam40" pkg="e_cam40_ros_driver" type="e_cam40_node" output="screen">

  	  <param name="device" type="str" value="/dev/video1" />

      <!-- <param name="fps" type="int" value="15" /> -->

      <!-- param name="left/camera_info_url" value="file://$(find tara_camera_driver)/calib/calib_left.yml"/ -->
      <!-- param name="right/camera_info_url" value="file://$(find tara_camera_driver)/calib/calib_right.yml"/ -->

      <param name="ir/camera_name" value="ir_cam"/>
      <param name="rgb/camera_name" value="rgb_cam"/>

      <param name="exposure" type="int" value="156" />
      <param name="brightness" type="int" value="10" />

      <param name="imageWidth" type="int" value="672" />
      <param name="imageHeight" type="int" value="380" />

      <!-- other possible image sizes:
                672x380
                1280x720
                1920x1080
                2688x1520
                -->

    </node>

  <!--/group-->

</launch>
